compiler/rustc_error_codes/src/error_codes/E0542.md

```diff
@@ -0,0 +1,48 @@
+The `since` value is missing in a stability attribute.
+
+
```

Comment:
The empty line is still here. :)

Reply:
Sorry, I'm still running the tests locally to confirm everything is fine. I'll push as soon as possible :).

Moreover I made a little change in the text to be consistent with the new decription you added.